packages/backend/src/Node.ts

```diff
@@ -177,22 +177,8 @@ export default class Node {
     return location ? [location.lat, location.lon, location.city] : null;
   }
 
-  public get average(): number {
-    let accounted = 0;
-    let sum = 0;
-
-    for (const time of this.blockTimes) {
-      if (time) {
-        accounted += 1;
-        sum += time;
-      }
-    }
-
-    if (accounted === 0) {
-      return 0;
-    }
-
-    return sum / accounted;
+  public get average(): Types.Milliseconds {
+    return blockAverage(this.blockTimes);
```

Comment:
`blockAverage` returns `Maybe`, typescript should complain here (though that should be fixed on the function itself, comment below).